apps/cowswap-frontend/src/modules/tradeQuote/utils/quoteUsingSameParameters.ts

```diff
@@ -45,7 +47,7 @@ function removeQuoteMetadata(appData: string | undefined): string | undefined {
 
   if (!decoded) return
 
-  const { metadata: fullMetadata, ...rest } = decoded
-  const { quote: _, ...metadata } = fullMetadata
+  const { metadata: fullMetadata, ...rest } = decoded as LatestAppDataDocVersion
```

Comment:
nit: maybe you want to do `isAppDataLatestVersion` that just check the property `version` matching `import { LATEST_APP_DATA_VERSION } from '@cowprotocol/app-data'`

Reply:
Removed that along with `hooks` omitting